.github/workflows/test-helm-charts.yml

```diff
@@ -113,3 +120,8 @@ jobs:
       - name: Run chart-testing (install)
         run: ct install --config .ci/ct.yaml
         if: steps.list-changed.outputs.changed == 'true'
+
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
```

Comment:
Should we add a timeout for this in case anyone forgot to check the status of the action?

Reply:
there is already a default timeout for tmate